packages/expo-screen-orientation/ios/EXScreenOrientation/EXScreenOrientationViewController.m

```diff
@@ -13,6 +13,15 @@ @interface EXScreenOrientationViewController ()
 
 @implementation EXScreenOrientationViewController
 
+- (instancetype)init
+{
+  if (self = [super init]) {
```

Comment:
Let's delegate to another initializer:

`return [self initWithDefaultScreenOrientationMask:UIInterfaceOrientationMaskPortrait]`